### DISCUSSION TITLE
brennt am anfang und ende zu stark ,,,habe das selbe problem auch mit m4 gibts eine lösung

### DISCUSSION THREAD

**Discussion by fude57** [arkypita/LaserGRBL#1130] [Q&A]

**Comment by BeatSlayer** [comment]
Be sure to enable "laser mode" by setting $32=1 in the GRBL configuration.

**Comment by Danotec** [comment]
Wenn Sie Einbrand an den Start und Endpunkten im Bild haben, versuchen Sie die Parameter $120 und $121 ( X und Y Beschleunigung der Motoren zu erhöhen.
Ich denke mit Werten von 1800-2500 mm/sek²  sollte das Problem gelöst sein.
________________________________________________
If you have brand at the start and end points in the picture, try increasing the parameters $ 120 and $ 121 (X and Y acceleration of the motors.
I think with values of 1800-2500 mm / sec² the problem should be solved.


Grüße!

**Comment by fude57** [comment]
Besten dank, werde ich ausprobieren
G rolf

Gesendet mit der blue News & E-Mail App

Am 2 November 2020 um 03:28 schrieb Dano <notifications@github.com>:

**Comment by Danotec** [comment]
Achten Sie darauf das Ihre Motoren und auch  die Motorhalter sowie Antriebselemente die Belastung auch auf dauer aushalten können, ggf. besser mit den Beschleunigungswerten etwas zu wenig als zu viel.

__________________________________________________________________________________________

Make sure that your motors and also the motor mounts as well as drive elements can withstand the load in the long term, if necessary with the acceleration values a little too little than too much.

**Comment by fude57** [comment]
Habe von 110mm sec auf 300 mm gestellt und problem ist gelöst, besten dank für tip 

Gesendet mit der blue News & E-Mail App

Am 2 November 2020 um 22:15 schrieb Dano <notifications@github.com>:
